### ISSUE TITLE
Migrate to Gtk only?

### ISSUE THREAD

**Issue by ararslan** [JuliaGraphics/Winston.jl#281]
Currently this package uses Tk by default with the option of using Gtk. However, the Tk package does not support precompilation, which means Winston similarly cannot support it. If we make the switch to Gtk, we'll get precompilation plus faster rendering. See discussion in https://github.com/JuliaGraphics/Winston.jl/pull/279 for some further details. What are your thoughts? cc @nolta in particular.

**Comment by nolta** [comment]
I'm ok with switching, but it seem premature, given [this problem](https://github.com/JuliaGraphics/Winston.jl/pull/279#issuecomment-315179759).

**Comment by ararslan** [comment]
Right, it would take some updating and maintenance up front, likely both here and Gtk.

**Comment by tknopp** [comment]
ping @ararslan, I agree that this should be done. Should I prepare a PR?

**Comment by ararslan** [comment]
If you'd be willing to do that, I think it's the right way to go!
